### PR TITLE
fixed typos

### PR DESCRIPTION
**Pull request by cdietrich**
fixed typos
Signed-off-by: Christian Dietrich <christian.dietrich@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 import com.google.inject.Inject;
 
 /**
- * Portable URIs are based on names and therefore are independent of the concrete file pathes and fuile names the of
+ * Portable URIs are based on names and therefore are independent of the concrete file paths and file names the of
```
Comment (tivervac) [inline org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java:40]: s/the of/of the/

Comment (cdietrich) [reply org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java]: argg that one left my spotting eye

File: org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java [hunk 2]
```diff
@@ -37,7 +37,7 @@
 import com.google.inject.Inject;
 
 /**
- * Portable URIs are based on names and therefore are independent of the concrete file pathes and fuile names the of
+ * Portable URIs are based on names and therefore are independent of the concrete file paths and file names of the
```
Comment (kthoms) [inline org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java:40]: independent _from_
"independent from" => separate from each ther
"independent of" => not influenced by each other

Comment (cdietrich) [reply org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java:40]: @kthoms i believe we mean the latter here

Comment (szarnekow) [reply org.eclipse.xtext/src/org/eclipse/xtext/resource/persistence/PortableURIs.java:40]: When in doubt, rephrase ;) `are based on names and do not depend on file paths ...`

**Review by szarnekow — APPROVED** [review]

**Pull request merged**